dnsapi/dns_ad.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env sh
+
+#
+#AD_API_KEY="sdfsdfsdfljlbjkljlkjsdfoiwje"
+
+#This is the Alwaysdata api wrapper for acme.sh
+
```

Comment:
Please add your github repository  link here. So that bugs can be reported there.
see: https://github.com/Neilpang/acme.sh/blob/master/dnsapi/dns_myapi.sh#L9